packages/common/src/state/originalStorageCache.ts

```diff
@@ -0,0 +1,57 @@
+import { bytesToUnprefixedHex } from '@ethereumjs/util'
+
+import type { Address } from '@ethereumjs/util'
```

Comment:
nbd, but just pointing out that there is a more concise syntax for this: 

```suggestion
import { type Address, bytesToUnprefixedHex } from '@ethereumjs/util'
```

Reply:
Doesn't / shouldn't the linter do this? :thinking:

Reply:
Yeah, I would like to stick to what the linter proposes (and my original version was what was proposed)